azext_edge/edge/providers/orchestration/permissions.py

```diff
@@ -62,29 +62,39 @@ class PermissionState(Enum):
     ActionUndefined = 3
 
 
+class PrincipalTypes(Enum):
+    # There are more types but keeping this short with respect to what we use for now
+    user = "User"
```

Comment:
Let's use all caps for enum names, here and moving forward, to conform better with PEP8.

In a future iteration we should re-align our enum types to subclass the types they are representing, like here being str i.e. `class PrincipalTypes(str, Enum):` which will give us a number of benefits.